staging/src/k8s.io/pod-security-admission/policy/registry.go

```diff
@@ -107,6 +110,23 @@ func validateChecks(checks []Check) error {
 			maxVersion = c.MinimumVersion
 		}
 	}
+	// Second pass to validate overrides.
+	for _, check := range checks {
+		for _, c := range check.Versions {
+			if len(c.OverrideCheckIDs) == 0 {
+				continue
+			}
+
+			if check.Level != api.LevelRestricted {
+				return fmt.Errorf("check %s: only restricted checks may set overrides", check.ID)
+			}
+			for _, override := range c.OverrideCheckIDs {
+				if overriddenLevel, ok := ids[override]; ok && overriddenLevel != api.LevelBaseline {
```

Comment:
should this error if an unknown overridecheckid is specified?

Reply:
See my comment on `TestValidChecks`

> I intentionally didn't put that in validateChecks because if you're customizing the list of checks it seems reasonable to use the overriding version without the overridden version (i.e. only the restricted version without the baseline version).